services/store.go

```diff
@@ -1050,6 +1030,89 @@ func decodeTransfer(height uint32, info *event.ExecuteNotify, evt *event.NotifyE
 	return xfer
 }
 
+func checkEvmEventLog(evt *event.NotifyEventInfo) (bool, *ctypes.StorageLog) {
+	ethLog, err := event.NotifyEventInfoToEvmLog(evt)
+	if err != nil {
+		return false, nil
+	}
+	return true, ethLog
+}
+func parseEvmOngTransferLog(ethLog *ctypes.StorageLog) (*transfer, error) {
+	ongLog := types2.Log{
+		Address: ethLog.Address,
+		Topics:  ethLog.Topics,
+		Data:    ethLog.Data,
+	}
+	if ongLog.Address.Hex() == ONG_ADDR {
+		parsed, _ := abi.JSON(strings.NewReader(ERC20ABI))
```

Comment:
Not sure if it has much of an impact, but perhaps for performance reasons, parse the ABI in a package `init` function and save it to a global `erc20ABI` variable?

Reply:
great,I will fix it.  this parsed should  in init function